configs/openSUSE/sysctl-whitelist.toml

```diff
@@ -137,31 +137,19 @@ digester = "shell"
 hash = "7ec5f6a48c24802a58ebe31c962c0fb63d5215f4562bb200111c6ef9df380a6a"
 
 [[FileDigestGroup]]
-packages = [
-    "kubernetes1.20-kubeadm",
-    "kubernetes1.21-kubeadm",
-    "kubernetes1.22-kubeadm",
-    "kubernetes1.23-kubeadm"
-]
+package = "glob:kubernetes1.2[0-3]-kubeadm"
 type = "sysctl"
-note = "enabled IPv4 forwarding system wide, same content for all kubeadm versions <= 1.23"
+note = "enables IPv4 forwarding system wide, same content for all kubeadm versions <= 1.23"
 bugs = ["bsc#1174722", "bsc#1209363", "bsc#1210951"]
 [[FileDigestGroup.digests]]
 path = "/usr/lib/sysctl.d/90-kubeadm.conf"
 digester = "shell"
 hash = "e2c2ac17097616ee184af9965776f83ad87dcf9e82ada5c8a3ea0f8371813fe8"
 
 [[FileDigestGroup]]
-packages = [
-    "kubernetes1.24-kubeadm",
-    "kubernetes1.25-kubeadm",
-    "kubernetes1.26-kubeadm",
-    "kubernetes1.27-kubeadm",
-    "kubernetes1.28-kubeadm",
-    "kubernetes1.29-kubeadm"
-]
+package = "glob:kubernetes1.[2-9][0-9]-kubeadm"
```

Comment:
This glob would match all `kubernetes1.20` to `1.23` already whitelisted in the previous `FileDigestGroup`. I assume this works because entries are matched in order?

How would we handle a third subsequent `FileDigestGroup` with a different hash starting from e.g. a `kubernetes1.32`? Maybe you can explain this to me privately

Reply:
Yeah this overmatching is a bit unfortunate but shouldn't hurt the algorithm.

The checker looks at all entries if it can find a combination where package name, path names and digests match.

In this instance this means that `kubernetes1.20 to 1.23` could now also ship the newer version (digest) of the sysctl files. An acceptable risk I guess.

Reply:
A bit unclean, but should be alright. Could it ever move backwards? That would be problematic

Reply:
Sounds good. Should there ever be a version of a whitelisted file which could be dangerous to go back to, we can take special care to avoid that.